examples/tracer/src/sdk_escape_hatch.py

```diff
@@ -11,9 +11,9 @@ def collect_payment(charge_id: str) -> str:
 @tracer.capture_lambda_handler
 def lambda_handler(event: dict, context: LambdaContext) -> str:
     charge_id = event.get("charge_id", "")
-    with tracer.provider.in_subsegment("## collect_payment") as subsegment:
-        subsegment.put_annotation(key="PaymentId", value=charge_id)
+    with tracer.provider.trace("## collect_payment") as span:
+        span.set_attribute(key="PaymentId", value=charge_id)
         ret = collect_payment(charge_id=charge_id)
-        subsegment.put_metadata(key="payment_response", value=ret)
+        span.set_attribute(key="payment_response", value=ret)
```

Comment:
Please keep the old example as it was to make sure it still works and maybe create a new one?